### DISCUSSION TITLE
Using DRNs and TVK/TVS to simulation mining

### DISCUSSION THREAD

**Discussion by RyanConway91** [MODFLOW-ORG/modflow6#2023] [Q&A]
I am using DRNs and TVK/TVS to simulate development and post-closure of a large open pit hard-rock mine. 

During mining, I am using drns to dewater the mine and tvk to increase the bedrock k in the mine to that it can be dewatered. After mining, I use TVS to set the whole mine to "water" (sy = 1.0, ss = compressibility of water). I don't do this during mining because if I did the mine would be dewatering this high storage material, which is not the case. 

My issue is that after using TVS to set the mine to "water", the GW inflow into the pit **increases**. At this point the pit is already at its maximum extent, so it should start to fill up with water and the GW inflow should decrease through time as the pit slowly reaches equilibrium with the surrounding GW system. My theory is the high "water" storage is the issues. The water sy is orders of magnitude higher than that of the rock, so it takes a lot more GW inflow to get a unit head rise in the pit than it did during dewatering. 

Does this theory make sense? 

If so, is there a way to use tvs to get my pit lake or do I need to use LAK? I did not not use LAK initially because the pit is also being backfilled post mining, which I could not figure out how to simulate using LAK.

**Comment by aprovost-usgs** [comment]
From: RyanConway91 ***@***.***>
Sent: Thursday, September 19, 2024 6:38 AM
To: MODFLOW-USGS/modflow6 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [MODFLOW-USGS/modflow6] Using DRNs and TVK/TVS to simulation mining (Discussion #2023)


 Ryan,

What you said about the inflow decreasing over time as the pit fills up, and it taking more inflow of water to effect a given rise in the water level in the pit after the specific yield is increased, does make sense.  However, the fact that a given rise requires more water does not necessarily mean there will be more flow - it does mean that for a given flow, the level will rise more slowly.

If I understand correctly, your concern is that after you use TVS to reset the storage properties in the pit to "water" at the end of mining, you observe an initial increase in the rate of inflow to the pit, which is unexpected.  How do you calculate inflow to the pit during mining and after mining?  At the end of the mining phase, the "inflow to the pit" involves water going to drains that represent the seepage face.  After that, when you invoke TVS, you presumably also deactivate the drains to stop dewatering, so the inflow to the pit will be from bedrock cells to pit cells, with nothing "diverted" by drains.  To help me understand and hopefully explain the increase in inflow, it would be helpful to know how you're reckoning the inflow right before and right after the end of mining.

-- Alden



 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



I am using DRNs and TVK/TVS to simulate development and post-closure of a large open pit hard-rock mine.

During mining, I am using drns to dewater the mine and tvk to increase the bedrock k in the mine to that it can be dewatered. After mining, I use TVS to set the whole mine to "water" (sy = 1.0, ss = compressibility of water). I don't do this during mining because if I did the mine would be dewatering this high storage material, which is not the case.

My issue is that after using TVS to set the mine to "water", the GW inflow into the pit increases. At this point the pit is already at its maximum extent, so it should start to fill up with water and the GW inflow should decrease through time as the pit slowly reaches equilibrium with the surrounding GW system. My theory is the high "water" storage is the issues. The water sy is orders of magnitude higher than that of the rock, so it takes a lot more GW inflow to get a unit head rise in the pit than it did during dewatering.

Does this theory make sense?

If so, is there a way to use tvs to get my pit lake or do I need to use LAK? I did not not use LAK initially because the pit is also being backfilled post mining, which I could not figure out how to simulate using LAK.

-

**Reply by RyanConway91** [reply]
I am using ZONEBUDGET to compute the mass balance within the pit and looking at the GWF-IN, Yes, DRNs are deactivated after mining. 
The increase in inflow post-mining (when the "water" tvs is activated) causes an increase in GWF-IN into the pit. I wouldn't call it a spike, its sustained for many  years but does start going down eventually. 

Agreed that the higher sy would take more water for a unit head rise, but that doesn't mean more water is available. Hence my confusion at the results. 

Thanks!

**Comment by aprovost-usgs** [comment]
Ryan,

Thanks for explaining how you're doing the inflows.  Since GWF-IN doesn't include drain flows, it seems that GWF-IN would be smaller when drains that represent the seepage face are handling much of the "inflow to the pit," and larger when you get rid of the drains and GWF-IN accounts for all of the inflow to the pit.  If you add the seepage-face drain flows to GWF-IN to get the "before" value for inflow, do you get something closer to the "after" value?

Btw I like how you phrased, "the higher sy would take more water for a unit head rise, but that doesn't mean more water is available."  Gets more directly to the point than the way I said it.

-- Alden

From: RyanConway91 ***@***.***>
Sent: Thursday, September 19, 2024 12:35 PM
To: MODFLOW-USGS/modflow6 ***@***.***>
Cc: Provost, Alden ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [MODFLOW-USGS/modflow6] Using DRNs and TVK/TVS to simulation mining (Discussion #2023)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



I am using ZONEBUDGET to compute the mass balance within the pit and looking at the GWF-IN, Yes, DRNs are deactivated after mining.
The increase in inflow post-mining (when the "water" tvs is activated) causes an increase in GWF-IN into the pit. I wouldn't call it a spike, its sustained for many years but does start going down eventually.

Agreed that the higher sy would take more water for a unit head rise, but that doesn't mean more water is available. Hence my confusion at the results.

Thanks!

-

**Reply by RyanConway91** [reply]
Wouldn't the DRN-OUT include GWF-IN? Doesn't water have to flow into the zone via GWF-IN before the drains can take it out?

Regardless, I figured out my issue, there was an error in my TVS/TVK setup with how I was representing backfilling. I did some checks and found that the total volume of sy in the zone increased during backfilling, which is obviously wrong. With this error (an increase in sy after mining) I imagine the heads in the pit zone went down, thus increasing the gradient and increasing GWF-IN.

**Comment by aprovost-usgs** [comment]
Ryan,

I was assuming the seepage-face drain cells were outside the "pit" zone, so you'd adding the flows exiting those drain cells to the groundwater flow entering the pit cells (GWF-IN) to get the total inflow to the pit.  You've included the seepage-face drain cells in the pit zone (under the reasonable assumption that any groundwater flow that enters those cells will exit through the drains, and won't escape to any neighboring non-drain cells).  In that case, GWF-IN should include all inflow to the pit.  Anyway, just wanted to make sure we were comparing apples to apples.

I agree that increasing Sy will drop the water level in the pit and increase the gradient initially.  That was my first thought when you described your problem, though I figured it would be a relatively minor effect assuming there's not much water in the pit at the end of mining.  Sounds like the bigger issue was a continued increase in Sy as backfilling progressed.  In any case, I'm glad you were able to track down the issue.

Best,

-- Alden

From: RyanConway91 ***@***.***>
Sent: Sunday, September 22, 2024 4:41 PM
To: MODFLOW-USGS/modflow6 ***@***.***>
Cc: Provost, Alden ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [MODFLOW-USGS/modflow6] Using DRNs and TVK/TVS to simulation mining (Discussion #2023)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



Wouldn't the DRN-OUT include GWF-IN? Doesn't water have to flow into the zone via GWF-IN before the drains can take it out?

Regardless, I figured out my issue, there was an error in my TVS/TVK setup with how I was representing backfilling. I did some checks and found that the total volume of sy in the zone increased during backfilling, which is obviously wrong. With this error (an increase in sy after mining) I imagine the heads in the pit zone went down, thus increasing the gradient and increasing GWF-IN.

-
